core/fokia/provisioner.py

```diff
@@ -492,6 +500,16 @@ def get_server_private_ip(self, server_id):
     """
     CHECK RESOURCES
     """
+    def check_flavor(vcpus, ram, disk):
+        """
+        Check if flavor with this resources allows creation.
```

Comment:
Can you please rewrite the description? It is difficult to understand what this method is supposed to do